doc/developers/contributing.rst

```diff
@@ -32,19 +32,12 @@ See :ref:`new_contributors` to get started.
     others of the Python Software Foundation:
     https://www.python.org/psf/codeofconduct/
 
-|
-
 
 In case you experience issues using this package, do not hesitate to submit a
 ticket to the
 `GitHub issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_. You are
 also welcome to post feature requests or pull requests.
 
-Governance
-----------
-The decision making process and governance structure of scikit-learn is laid
```

Comment:
You want this removed?

Reply:
I added a link at the top (not worth a whole section IMO)